include/wasm/WasmModule.h

```diff
@@ -36,14 +35,16 @@
 namespace wasm {
     class WasmModule {
     public:
+        virtual ~WasmModule();
+
         // ----- Module lifecycle -----
-        virtual void bindToFunction(const message::Message &msg) = 0;
+        virtual void bindToFunction(const message::Message &msg);
```

Comment:
I've provided stubbed implementations of these so every subclass doesn't need to have a load of empty functions.